### ISSUE TITLE
Is there a way to listen to the close event?

### ISSUE THREAD

**Issue by jsrilapan** [dfahlander/Dexie.js#1211]
Hello there, I'm looking for a way to catch a close event when the user explicitly deletes the database. 
For example, user clears browsing data or just deletes database from the Chrome dev tools.

The native IDBDatabase interface has this [onclose](https://developer.mozilla.org/en-US/docs/Web/API/IDBDatabase/onclose) event handler. It will fired when the situations above occurred so I was wondering if there is a possible way that Dexie can listen or expose that event too?

Thanks in advance.

**Comment by dfahlander** [comment]
Dexie should definitely have its own on('close') API (PR welcome!) but for now you can do the following:

```js
const db = new Dexie("dbname");
db.on('ready, () => {
  db.backendDB().onclose = event => {
    myAppShowAlert('The database "' + db.name + '" has unexpectedly closed.');
  };
});
```

https://dexie.org/docs/Dexie/Dexie.backendDB()

**Comment by jsrilapan** [comment]
Thanks for your respond, I will make a PR when it's ready.

**Comment by dfahlander** [comment]
Solved in #1212
